Sources/PointFree/Pricing.swift

```diff
@@ -71,14 +71,71 @@ private let pricingView = View<Prelude.Unit> { _ in
                   ])
                 ]),
 
+              ]),
+            form([action("/subscribe"), id("payment-form"), method(.post)], [
+              input([name("plan"), type(.hidden), value(Stripe.Plan.Id.monthly.rawValue)]),
+              input([name("token"), type(.hidden)]),
+              div([id("card-element"), data("stripe-key", AppEnvironment.current.envVars.stripe.publishableKey)], []),
+              div([id("card-errors"), role(.alert)], []),
+              button(["Submit Payment"])
               ])
             ]),
 
           ]),
         ]
-        <> footerView.view(unit))
-      ])
+        <> footerView.view(unit)
+        <> [
+          script([src("https://js.stripe.com/v3/")]),
+          script(
+            """
+            var apiKey = document.getElementById('card-element').dataset.stripeKey;
```

Comment:
Out script tag takes a `StaticString`, so things like the API key, here, can't be interpolated. That's potentially a good thing, though was a gotcha!

Reply:
lol yeah. what do you think about having an `unsafeString` function that allows unencoded data through? seems no matter what we need an escape hatch unless we attempt Swift.js :/

Reply:
also whatdya think about moving this into its own string (and maybe file) like we did for svg? to keep this view light.